### DISCUSSION TITLE
Why have seperate relational database?

### DISCUSSION THREAD

**Discussion by pallabez** [Real-Dev-Squad/website-goals#88] [General]
## Requirement 
- The service should be reusable without any RDS dependency. 
- JSON:API specs needs to be followed.

### So why seperate database

- To avoid coupling with RDS
- Only partial JSON:API specs can be followed if using non relational database.
- Low effort on setting up the api as python [framework](https://django-rest-framework-json-api.readthedocs.io/en/stable/) is available.
- No RDS related security key is needed 

### Disadvantage 

- Authentication flow is complicated (but have been [planned](https://github.com/Real-Dev-Squad/website-goals/discussions/67) and partially completed)

**Comment by pallabez** [comment]
### If we were to use existing database then how much time or effort we need to make it right?

- @ivinayakg Can you please look into it? Take help from Prakash when needed.

**Reply by ivinayakg** [reply]
By existing database you are referring to firestore which is the current configuration, right?

**Reply by pallabez** [reply]
Yes
